src/components/templates/AddStory.js

```diff
@@ -57,14 +58,21 @@ const AddStory = ({ toggleModal }) => {
     localStorage.setItem(`story-${name}`, value);
   };
 
+  const button = (
+    <Button
+      type="button"
+      text="Submit Story"
+      clickHandler={addNewStory}
+      variant="primary"
+    />
+  );
+
```

Comment:
Passing in this button that acts as the submit button instead of having the form handle the submit allows us to prevent the user from submitting the form (whatever that form may be) via typing Enter on the keyboard unless they purposely tab to the submit button and click enter.

This allows us to save the user from accidentally submitting important modals by typing enter in a field when they might not have meant to submit the form from such action.